src/silx/math/combo.pyx

```diff
@@ -27,12 +27,12 @@ For now it provides min/max (and optionally positive min) and indices
 of first occurrences (i.e., argmin/argmax) in a single pass.
 """
 
-__authors__ = ["T. Vincent"]
+__authors__ = ["T. Vincent", "Jérôme Kieffer"]
 __license__ = "MIT"
-__date__ = "24/04/2018"
+__date__ = "09/09/2025"
 
 cimport cython
-from .math_compatibility cimport isnan, isfinite, INFINITY
```

Comment:
`math_compatibility.pxd` is no longer needed on Windows?
Can be removed (in another PR)?

Reply:
I don't use it in pyFAI nor fabio and it works great for ages ... So yes, it can probably be removed.

Reply:
-> #4398